templates/home/home-en.html

```diff
@@ -16,13 +16,8 @@ <h1 property="name" id="wb-cont">Canada.ca</h1>
 	</div>
 	<section class="gc-featured-link">
 		<div class="container">
-			<div class="row">
-				<div class="col-xs-12">
-					<h2 class="wb-inv">Featured</h2>
-					<p><a class="stretched-link" href="#">[Promotion title]</a></p>
-					<p>Long description [Optional]</a></p>
-				</div>
-			</div>
+			<h2 class="wb-inv">Spotlight on</h2>
+			<p><a class="stretched-link" href="#">[Promotion title]</a></p>
```

Comment:
That is great, I suggest to keep the example with the optional long description for this working example.